EFCore.NamingConventions/Internal/LowerCaseFirstCharacterNameRewriter.cs

```diff
@@ -0,0 +1,18 @@
+using System.Globalization;
+
+namespace EFCore.NamingConventions.Internal
+{
+    public class LowerCaseFirstCharacterNameRewriter : INameRewriter
+    {
+        private readonly CultureInfo _culture;
+
+        public LowerCaseFirstCharacterNameRewriter(CultureInfo culture) => _culture = culture;
+        public string RewriteName(string name)
```

Comment:
nit: can you please rewrite this method as an expression-bodied method (with the ternary operator)?

Reply:
ok

Reply:
done